src/hooks/i18n.ts

```diff
@@ -0,0 +1,93 @@
+import React from "react"
```

Comment:
It's a nice little exercise to write something like this ourselves, but I think we can just use the actual `react-i18next` and just shove the translations in it. This way we immediately have support for parameter substitution, components in translations, etc. 😉